public/Lab-Files/Lesson04/Challenges/Team-Challenge-1/css/styles.css

```diff
@@ -0,0 +1,32 @@
+header{
+    margin:auto;
+    text-align:center;
+    
+}
+
+.one{
+    display: block;
+    max-width:400px;
+    max-height:250px;
+    width: 600px;
+    height: auto;
+    position:relative;
+}
+
+.left{
+    background-color:#F7F7F7;
+    float:left;
+    
+}
+
+.right{
+    float:right;
+    width:40%;
+}
+
+
+/*This controls the size of the pictures on the right*/
+.rightside {
+    width:100px;
+    height:100px;
```

Comment:
In the instructions it mentioned making images responsive using percentages. ```set image max-width property to 100% and height property to auto```. If you use pixels, it will just be a static size.